### PR TITLE
(feature) Added grid and did linting changes to README.md

### PR DESCRIPTION
**Pull request by prasidh-agg**
This has been done using tables in markdown, there is no way as such to stylise/colorise or customise them so have gone with simple approach. This looks best in light mode as most icons are on the darker side.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,49 +2,40 @@
...
--   [Zustand](https://github.com/pmndrs/zustand)
--   [TailwindCSS](https://tailwindcss.com/)
--   [Nodejs](https://nodejs.org/en/)
--   [GraphQL](https://graphql.org/)
--   [GiraphQL - SchemaBuilder](https://giraphql.com/)
--   [Socket.IO](https://socket.io/)
--   [Prisma](https://www.prisma.io/)
--   [PostgresSQL](https://www.postgresql.org/)
--   [Docker](https://www.docker.com/)
+| [![TypeScript](images/typescript.png)](https://www.typescriptlang.org/)| [![TypeScript](images/nextjs.png)](https://nextjs.org/)| [![TypeScript](images/react.png)](https://reactjs.org) | [![TypeScript](images/apollo.png)](https://www.apollographql.com/) |
```
Comment (FourLineCode) [inline README.md:15]: all the images have same alt-text .. this needs to be changed

File: README.md [hunk 2]
```diff
@@ -2,49 +2,40 @@
...
+|---|---|---|---|
+| [![TypeScript](images/tailwind.png)](https://tailwindcss.com/) | [![TypeScript](images/nodejs.png)](https://nodejs.org/en/)| [![TypeScript](images/graphql.png)](https://graphql.org/) | [![TypeScript](images/socket.png)](https://socket.io) |
```
Comment (FourLineCode) [inline README.md:17]: you can use `` tag with html attributes for styling

**Review by FourLineCode — CHANGES_REQUESTED** [review]
Good job so far .. :100: 

but you can actually use html tags in readme for images with source and style them using html attributes ..
maybe try to find images with background instead of transparent png for better dark mode? also get the GiraphQL logo from their website giraphql.com
also currently all the images have same alt-text, but i dont think that is relevant anymore

Good luck :+1:

**Comment by prasidh-agg** [comment]
cool, thanks for the feedback, let me try some more stuff in that case

File: README.md (hunk 2 above)
Comment (prasidh-agg) [reply README.md:17]: we can only control width and height there and that wont be really helpful in this case, tbh.

**Pull request closed without merging**